lms/djangoapps/teams/tests/test_views.py

```diff
@@ -675,7 +675,7 @@ def build_team_data(
 
     def post_create_team(self, expected_status=200, data=None, **kwargs):
         """Posts data to the team creation endpoint. Verifies expected_status."""
-        return self.make_call(reverse('teams_list'), expected_status, 'post', data, **kwargs)
+        return self.make_call(reverse('teams_list'), expected_status, 'post', data, topic_id='topic_0', **kwargs)
```

Comment:
hmm, what is this doing?

Reply:
Actually, given I updated to "build team", it may not be necessary.  The initial intention, as I'm still coming up to speed in Python, was to add the missing topic_id ... I'm trying the test suite now without it.

Reply:
Removed repeated argument.